.github/auto-assignees.yaml

```diff
@@ -20,4 +22,4 @@ skipKeywords:
   - wip
   - DO NOT MERGE
 
-numberOfReviewers: 1
+numberOfReviewers: 2
```

Comment:
I don't think this should be changed. `1` means that one reviewer from each group will be chosen (2 groups, so 2 reviewers). Increasing this to 2 will mean 4 reviewers are pinged from each PR.

I think you want to increase the number of approvers needed to merge a PR, that needs to happen through the project settings itself.

Reply:
thx, updated.